.pre-commit-config.yaml

```
@@ -13,6 +13,16 @@ ci:
 repos:
   - repo: local
     hooks:
+      - id: build-syntaxes
+        name: Check the syntax files are in sync
+        entry: python scripts/build-syntaxes
```

Since you change the script to Python, why not make it `.py`?
```suggestion
        entry: python -m scripts.build_syntaxes
```